jest.config.js

```diff
@@ -12,16 +12,22 @@ module.exports = {
   testMatch: ['**/*.test.(ts|js)'],
   testEnvironment: 'node',
   preset: 'ts-jest',
-  collectCoverageFrom: ['./src/**/*.ts', '!./src/migration/**', '!./src/tests/**', '!./src/index.ts', '!./src/slack/**', '!./src/discord/index.ts'],
+  collectCoverageFrom: [
+    './src/**/*.ts',
+    '!./src/migration/**',
+    '!./src/tests/**',
+    '!./src/index.ts',
+    '!./src/slack/**',
+    '!./src/discord/index.ts',
+    '!./src/logger.ts',
+    '!./src/app.ts',
+    '!./src/StringDictionary.ts',
+    '!./src/api/admin.ts',
+    '!./src/api/index.ts',
```

Comment:
Is the eventual goal to get this down to as few files ignored as possible?

Reply:
Some of them, yes. I don't see value in testing some of the files like the logger.